src/jit/compiler.cpp

```diff
@@ -546,13 +546,9 @@ var_types    Compiler::argOrReturnTypeForStruct(unsigned size, CORINFO_CLASS_HAN
             if (size <= MAX_RET_MULTIREG_BYTES)
             {
 #ifdef _TARGET_ARM64_
-                assert(size > TARGET_POINTER_SIZE);
-
-                // For structs that are 9 to 16 bytes in size set useType to TYP_STRUCT, 
-                // as this means a 9-16 byte struct value in two registers
-                //
-                useType = TYP_STRUCT;
```

Comment:
Till we implement multi-reg return of structs, i thought the useType will be TYP_STRUCT.  Why are we deleting useType = TYP_STRUCT assignment?